src/app/modules/item/http-services/get-item-by-id.service.ts

```diff
@@ -34,6 +34,10 @@ export const itemDecoder = pipe(
     noScore: D.boolean,
     titleBarVisible: D.boolean,
     fullScreen: D.literal('forceYes','forceNo','default'),
+    allowsMultipleAttempts: D.boolean,
+    duration: D.nullable(D.string),
+    enteringTimeMin: D.string,
+    enteringTimeMax: D.string,
```

Comment:
you should use the date decoder to get a date directly (see https://github.com/France-ioi/AlgoreaFrontend/blob/master/src/app/shared/helpers/decoders.ts, might require to get the latest version of master for that)

Reply:
Fixed